R/append-dataset.R

```diff
@@ -5,14 +5,16 @@
 #' `dataset2` is not a Crunch dataset, it will be uploaded as a new
 #' dataset before appending. If it is a CrunchDataset, it may be subsetted with
 #' a filter expression on the rows and a selection of variables on the columns.
-#' @param autorollback logical: If the append fails, revert the dataset back
+#' @param autorollback Deprecated. This option no longer does anything.
+#' logical: If the append fails, revert the dataset back
 #' to its state before attempting to append? Default is \code{TRUE}, and you
 #' probably won't want to change that.
 #' @return `dataset1`, updated with `dataset2`, potentially filtered on rows and
 #' variables, appended to it.
 #' @export
-appendDataset <- function (dataset1, dataset2, autorollback=TRUE) {
+appendDataset <- function (dataset1, dataset2, autorollback) {
     stopifnot(is.dataset(dataset1))
+
```

Comment:
```
if (!missing(autorollback)) {
    warning("The 'autorollback' argument is deprecated and has no effect", call.=FALSE)
}
```

and test for that warning.

Reply:
This is done, but GitHub's magic for hiding comments that are addressed was foiled by it being a line below.